src/gatsby-remark-download-images.js

```diff
@@ -0,0 +1,52 @@
+const visit = require('unist-util-visit')
+const visitWithParents = require(`unist-util-visit-parents`)
+const path = require("path");
+const url = require('url');
+const fs = require('fs');
+const fsExtra = require("fs-extra")
+const slugify = require('slugify')
+var mime = require('mime-types');
```

Comment:
I think `import` works in node 12 btw

Reply:
I saw this: https://stackoverflow.com/questions/39436322/node-js-syntaxerror-unexpected-token-import

" Since Node 12, you can use either the .mjs extension, or set "type": "module" in your package.json. And you need to run node with the --experimental-modules flag."

But I saw that Gatsby could have problem with the experimental flag.

Reply:
Maybe Babel? I will search more

Reply:
Maybe it's because I've been using TypeScript a lot with node for the past year so I got used to imports :D

nevermind then 👍